delphi-checks/src/main/java/au/com/integradev/delphi/checks/PointerNameCheck.java

```diff
@@ -31,6 +34,15 @@
 @Rule(key = "PointerName")
 public class PointerNameCheck extends DelphiCheck {
   private static final String MESSAGE = "Rename this type to match the expected naming convention.";
+  private static final String POINTER_PREFIX = "P";
+  private static final String EXTENDED_TYPE_PREFIXES = "T,E,I";
+
+  private List<String> extendedTypePrefixesList;
+
+  @Override
+  public void start(DelphiCheckContext context) {
+    extendedTypePrefixesList = Splitter.on(',').trimResults().splitToList(EXTENDED_TYPE_PREFIXES);
+  }
```

Comment:
This can be done more concisely by constructing the list directly as the static final variable with `List.of`:

```suggestion
  private static final List<String> EXTENDED_TYPE_PREFIXES = List.of("T", "E", "I");
```